frontend/app/forms/MemberForm.scala

```diff
@@ -108,7 +108,7 @@ object MemberForm {
     lazy val zuoraAccountAddress = billingAddress.getOrElse(deliveryAddress)
   }
 
-  case class FeedbackForm(category: String, page: String, feedback: String, name: String, email: String)
+  case class FeedbackForm(category: FeedbackType, page: String, feedback: String, name: String, email: String)
```

Comment:
I realise this was already here, but I it looks to me like `FeedbackForm` doesn't belong in `frontend/app/forms/MemberForm.scala` - should go in it's own file, especially now it's expanding with the `feedbackTypeFormatter`